app/uk/gov/hmrc/brm/utils/CommonUtil.scala

```diff
@@ -18,93 +18,62 @@ package uk.gov.hmrc.brm.utils
 
 import org.joda.time.LocalDate
 import play.api.http.HeaderNames
-import play.api.libs.json.{JsValue, Json}
+import play.api.libs.json.JsValue
 import play.api.mvc.{Controller, Request, Result}
 import uk.gov.hmrc.brm.config.BrmConfig
-import uk.gov.hmrc.brm.implicits.Implicits._
 import uk.gov.hmrc.brm.models.brm.Payload
-import uk.gov.hmrc.brm.utils.BrmLogger._
-import uk.gov.hmrc.play.http._
 
 import scala.util.matching.Regex
 import scala.util.matching.Regex.Match
 
-trait ControllerUtil {
-  self: Controller =>
+trait ControllerUtil extends Controller with BrmException {
```

Comment:
The BirthEventsController now extends the BRMBaseController (previously ControllerUtil) I've also split it into it's own file.